requirements-data_and_models.txt

```diff
@@ -18,10 +18,7 @@
 PyYAML==5.4.1
 dvc[ssh]==2.0.17
 https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.0.0/en_core_web_sm-3.0.0.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_bc5cdr_md-0.4.0.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_bionlp13cg_md-0.4.0.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_craft_md-0.4.0.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_jnlpba_md-0.4.0.tar.gz
+https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_core_sci_lg-0.4.0.tar.gz
```

Comment:
Why do we need this one ? Is it because we use it for testing purposes ?

Reply:
It was to anticipate #389 and to already have `en-core-sci-lg` installed on the `bbs-base` image before #389 gets merged. But now I should actually move it to `requirements.txt` since that's where it's ended up in #389

Reply:
I just actually moved it to make sure it's the same as in #389 (71c42a3)